front/js/getservice.html

```diff
@@ -0,0 +1,6 @@
+<html>
+    <head>
+        <script src="service.js"></script>
+        <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    </head>
+</html>
```

Comment:
Add a `<body></body>` and a `<script></script>` tag with an example code that demonstrate how to use the service. Otherwise this page is useless.

For instance : 
```html
<body>
<script>
  // Here is an example of how to use the service
  const scratchyApi = new ScratchyService("http://localhost:5000/api");
  api.fetchRooms().then( rooms => console.log(rooms));
</script>
</body>
```